test/compile.jl

```diff
@@ -578,27 +578,23 @@ let
             end
             """)
 
-        @sync for wid in test_workers
-            @async remotecall_fetch(Core.eval, wid, Base, quote
-                unshift!(LOAD_PATH, $load_path)
-                unshift!(LOAD_CACHE_PATH, $load_cache_path)
-            end)
+        @everywhere test_workers begin
```

Comment:
this is a pretty indirect test, could use something more specific in the distributed tests